README.md

```diff
@@ -432,6 +433,79 @@ Example 3:
 
 _________________________
 
+
+### qLoRA Tuning Example
+
+This method is similar to LoRA Tuning, but the base model is a quantized model.
+Set `peft_method` to `"lora"`. You can pass any of LoraConfig, see section on [LoRA Example](#lora-tuning-example).
```

Comment:
Would be good to note the additional qlora flags first and then note that the lora params are the same. Would be useful to have more details on what the LoRA quantization config flags mean and what kernels are supported. Right now only triton is supported